prow/istio-integ-race-k8s-tests.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
```

Comment:
You should reuse e2e-suite.sh, do not recreate the logic

Reply:
Done: changed to reuse e2e-suite.sh.